asset-bundle-converter/Assets/AssetBundleConverter/Editor/ExportFBXAssetBundles.cs

```diff
@@ -0,0 +1,212 @@
+using System;
+using UnityEngine;
+using UnityEditor;
+using System.IO;
+using System.Collections.Generic;
+using System.Linq;
+using DCL.ABConverter;
+using DCL.Shaders;
+using UnityEngine.Experimental.Rendering;
+using UnityEngine.Rendering;
+using Object = UnityEngine.Object;
+using Random = UnityEngine.Random;
+
+public class ExportFBXAssetBundles : MonoBehaviour
```

Comment:
Can we add this to a namespace similar to this one? 
https://github.dev/decentraland/asset-bundle-converter/blob/main/asset-bundle-converter/Assets/AssetBundleConverter/SceneClient.cs

Reply:
Done!